src/juju/reducers.js

```diff
@@ -2,21 +2,31 @@ import produce from "immer";
 
 import { actionsList } from "./actions";
 
-export default function jujuReducers(state = {}, action) {
-  return produce(state, draftState => {
+export default produce(
+  (draftState, action) => {
+    // The below is required because CRA ignores the eslint configuration file
+    // during development and adds warnings in the dev console.
+    // eslint-disable-next-line default-case
```

Comment:
I'm open to suggestions on how to avoid those warnings without resorting to weird hacks like this.

Reply:
Add a blank default?

```
      case actionsList.updateModelStatus:
        const payload = action.payload;
        draftState.models.items[payload.modelUUID].status = payload.status;
        break;
      default: // do nothing.
```

Reply:
It was then complaining about an empty default haha.

Reply:
I could then add a console log and then an exception for that 😆